workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/views/static_report.py

```diff
@@ -368,6 +363,23 @@ def render_hovmollers(metadata, diagnostics):
     )
 
 
+def render_horizontal_maps(metadata, diagnostics, metrics):
```

Comment:
"horizontal_map" is inconsistent with the name of the plotting routine "plot_cube".

Reply:
Consolidated around `cubed_sphere_map`